doc/appendix-de-kasssichv/appendix-de-kasssichv.md

```diff
@@ -94,12 +94,92 @@ The ftReceiptCase indicates the receipt type and defines how it should be proces
+| `0x4445000000000015` | "catalogue services special"<br />For processing, see (0x4445000000000004)                 | 0-                  |
+| `0x4445000000000016` | "catalogue services zero"<br />For processing, see (0x4445000000000005)                    | 0-                  |
+| `0x4445000000000017` | "own consumption discounted-1"<br />For processing, see (0x4445000000000001)               | 0-                  |
+| `0x4445000000000018` | "own consumption discounted-2"<br />For processing, see (0x4445000000000002)               | 0-                  |
+| `0x4445000000000019` | "own consumption normal"<br />For processing, see (0x4445000000000003)                     | 0-                  |
+| `0x444500000000001A` | "own consumption special"<br />For processing, see (0x4445000000000004)                    | 0-                  |
+| `0x444500000000001B` | "own consumption zero"<br />For processing, see (0x4445000000000005)                       | 0-                  |
+| `0x444500000000001C` | "down payment discounted-1"<br />For processing, see (0x4445000000000001)                  | 0-                  |
+| `0x444500000000001D` | "down payment discounted-2"<br />For processing, see (0x4445000000000002)                  | 0-                  |
+| `0x444500000000001E` | "down payment normal"<br />For processing, see (0x4445000000000003)                        | 0-                  |
+| `0x444500000000001F` | "down payment special"<br />For processing, see (0x4445000000000004)                       | 0-                  |
+| `0x4445000000000020` | "down payment zero"<br />For processing, see (0x4445000000000005)                          | 0-                  |
+| `0x4445000000000021` | "account of a third party/ third party name/ collection"<br />For processing, see (0x4445000000000007)                                                                                                                                                                                        | 0-                  |
+| `0x4445000000000022` | "obligation with signature requirement"<br />Obligations are to be equalized with pay items. If however, it is for technical reasons necessary to transfer obligations in the charge items block, then this code should be used for obligations with German law requirement. The gross amount due is recorded in the signature field, set zero, according to the German law.<br />For example, a receipt for a voucher issuance, for which the voucher is indicated as item in the charge items block and the corresponding cash amount is indicated in the pay items block.<br />An example for this would be a voucher intake via charge items block, or a payment of an outgoing invoice.                                                                                                                                                                                                 | 0-                  |
```

Comment:
there is no separation between signature required or not in germany. so there is maybe only one obligation item

Reply:
unified